src/mhz16/mhz16.cxx

```diff
@@ -37,12 +37,24 @@ MHZ16::MHZ16(int uart)
 {
```

Comment:
These commits regarding mhz16 should be in a separate PR.

Reply:
I think I have successfully rebased the other PR I will do hopefully the same for this one  soi'll close this one and open a new one .
You are right my mistake is to have used my dev branch for PRs I think the cleanest way is to cretae a temporary for each PR.
I'll cherrypick this commit in a separate branch